Made grammar/word-choice updates 🎓

EthanThatOneKid opened the pull request:
Here is my best effort as far as professional language and word-choice goes. Please feel free to suggest alternative wordings or grammar corrections.

EthanThatOneKid committed:
Made grammar/word-choice updates on all text shown throughout the site.

Here is my best effort as far as professional language and word-choice goes. Please feel free to suggest alternative wordings or grammar corrections.

diamondburned commented on src/components/index/why-join.svelte:
```diff
@@ -7,14 +7,19 @@
+      is a community that encourages growth, friendship, and technical development.
+      We prepare each other to overcome future personal, academic, and professional
+      challenges that some may face down the road. By joining
+      <span class="brand-em">acm<span class="brand-em">CSUF</span></span>,
```
The "CSUF" part seems inconsistent with what's 4 lines above.

diamondburned commented on src/routes/about.svelte:
```diff
@@ -8,25 +8,30 @@
   <h1 slot="title">about</h1>
   <h2 slot="headline">
-    what is acm<span class="brand-blue">CSUF</span>
+    What is acm<span class="brand-blue">CSUF</span>
```
This part doesn't have a question mark, which seems inconsistent with what's above.

diamondburned replied:
Shouldn't "acm" have `brand-em` here?

diamondburned commented on src/routes/connect.svelte:
```diff
@@ -11,29 +11,36 @@
+  <h2 slot="headline">Follow us!</h2>
   <p slot="text">
     The best way to keep up with what's going on in the world of
     <span class="brand-bold">acm<span class="brand-blue">CSUF</span></span>
```
Shouldn't "acm" have `brand-em` here instead as well?

diamondburned commented on src/routes/connect.svelte:
```diff
@@ -11,29 +11,36 @@
-    and to getback to you! <span class="brand-bold"
-      >acm<span class="brand-blue">CSUF</span></span
+    We are excited to take your messages and ideas into consideration! <span
+      class="brand-bold">acm<span class="brand-blue">CSUF</span></span
```
Shouldn't "acm" have `brand-em` here instead?

EthanThatOneKid committed:
Addressing review @diamondburned's initial review of this branch

Addressing review <https://github.com/EthanThatOneKid/acmcsuf.com/pull/107#pullrequestreview-650061470>.

EthanThatOneKid committed:
Update event-carousel.svelte

Capitalized "T" in "this".

samuelsandoval1 reviewed (commented):
Looks good. I believe acmCreate created the typography to be lowercase at the beginning of each sentence/header. So if you could keep to that that would be great. If mike says this way is fine then this is ok then!

samuelsandoval1 reviewed (changes requested):
Make first letter lowercased

EthanThatOneKid committed:
Adjusted all of the upper/lowercasing of all heading elements.

In every page, I made sure that the capitalization was as intended.

EthanThatOneKid committed:
Merge branch 'main' into fix/grammar-overhaul

EthanThatOneKid committed:
Removed duplicate officer instances on about page.

samuelsandoval1 reviewed (approved):
LGTM!
Make sure to push to production
